doc/interop-test-descriptions.md

```diff
@@ -293,16 +297,14 @@ Procedure:
 
 Asserts:
 * call was successful
-* received SimpleResponse.username equals FLAGS_default_service_account
-* received SimpleResponse.oauth_scope is in FLAGS_oauth_scope
+* received SimpleResponse.username equals --default_service_account
+* received SimpleResponse.oauth_scope is in --oauth_scope
```

Comment:
What does "is in" mean? Why isn't it just equals? If ```--oauth_scope``` is a list, then it needs to be given a format (such as comma-separated).